tests/foreman/ui/test_discoveredhost.py

```diff
@@ -100,13 +100,11 @@ def _is_host_reachable(host, retries=12, iteration_sleep=5, expect_reachable=Tru
     if not expect_reachable:
         operator = '||'
     cmd = 'for i in {{1..{0}}}; do ping -c1 {1} {2} exit 0; sleep {3}; done; exit 1'
-    result = ssh.command(
-        cmd.format(retries, host, operator, iteration_sleep), connection_timeout=30
-    )
+    result = ssh.command(cmd.format(retries, host, operator, iteration_sleep))
```

Comment:
self.execute is missing here, was it intentional

Reply:
execute isn't possible here since we are just passed a host address from the discovered host values. if that is something that is desired, then the component owner can do a larger refactor of this module